public/app/features/explore/MetaInfoText.test.tsx

```diff
@@ -1,34 +1,26 @@
 import React from 'react';
-import { shallow, render } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import { MetaInfoText, MetaItemProps } from './MetaInfoText';
 describe('MetaInfoText', () => {
-  it('should render component', () => {
+  it('should render component and items', () => {
     const items: MetaItemProps[] = [
       { label: 'label', value: 'value' },
       { label: 'label2', value: 'value2' },
     ];
-    const wrapper = shallow(<MetaInfoText metaItems={items} />);
-    expect(wrapper).toMatchSnapshot();
-  });
 
-  it('should render items', () => {
-    const items: MetaItemProps[] = [
-      { label: 'label', value: 'value' },
-      { label: 'label2', value: 'value2' },
-    ];
-
-    const wrapper = render(<MetaInfoText metaItems={items} />);
-    expect(wrapper.find('label')).toBeTruthy();
-    expect(wrapper.find('value')).toBeTruthy();
-    expect(wrapper.find('label2')).toBeTruthy();
-    expect(wrapper.find('value2')).toBeTruthy();
+    render(<MetaInfoText metaItems={items} />);
+    expect(screen.getAllByTestId('meta-info-text')).toHaveLength(1);
+    expect(screen.getAllByTestId('meta-info-text-item')).toHaveLength(2);
+    expect(screen.getByText('label:')).toBeInTheDocument();
+    expect(screen.getByText('label2:')).toBeInTheDocument();
+    expect(screen.getByText('value')).toBeInTheDocument();
```

Comment:
this will match also `value2` as `getByText` performs a case sensitive partial match, you may want to use:

```suggestion
    expect(screen.getByText(/^value$/)).toBeInTheDocument();
```

Reply:
🤦 absolutely, good catch!